include/CppUTest/CppUTestConfig.h

```diff
@@ -216,8 +218,6 @@ typedef struct { char dummy[16]; } cpputest_ulonglong;
 typedef struct { char dummy[8]; } cpputest_longlong;
 typedef struct { char dummy[8]; } cpputest_ulonglong;
 #endif
-#define CPPUTEST_LONGLONG_DEFAULT cpputest_longlong()
-#define CPPUTEST_ULONGLONG_DEFAULT cpputest_ulonglong()
```

Comment:
Ok, got the non-long-long support tests back and think I fixed the default value problem for mocks probably too. Still didn't do CMake, so expecting the build to still fail.